.github/workflows/release.yml

```diff
@@ -9,15 +9,15 @@ on:
   workflow_dispatch:
     inputs:
       version_bump:
-        description: 'Version bump type'
+        description: "Version bump type"
```

Comment:
Were these changes done automatically by some auto-formatter, or did you change this intentionally? (Here and below, and in `.golangci.yml`.)

I'd suggest to either drop them, or, if you do find them important, to do them in a separate "cleanup" commit. Just to reduce the diff noise in this commit which is already so large.